<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMyWpsClass</class>
 <widget class="QMainWindow" name="QtMyWpsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMyWps</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="symbolComboBox">
          <item>
           <property name="text">
            <string>标准</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(●)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(○)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(□)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(1.2.3.)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(a.b.c.)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(A.B.C.)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(ⅰ.ⅱ.ⅲ)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>项目符号(Ⅰ.Ⅱ.Ⅲ.)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="fontSizeComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QMdiArea" name="mdiArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="saveAsAction"/>
    <addaction name="pintAction"/>
    <addaction name="pintPreviewAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="EditMenu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
   </widget>
   <widget class="QMenu" name="FormatMenu">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="fontMenu">
     <property name="title">
      <string>字体(&amp;O)</string>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underLineAction"/>
    </widget>
    <widget class="QMenu" name="paragraphMenu">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="leftAlignAction"/>
     <addaction name="rightAlignAction"/>
     <addaction name="centerAction"/>
     <addaction name="justifyAction"/>
    </widget>
    <addaction name="fontMenu"/>
    <addaction name="paragraphMenu"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>窗体(&amp;W)</string>
    </property>
    <addaction name="closeAtion"/>
    <addaction name="closeAllAction"/>
    <addaction name="titleAction"/>
    <addaction name="cascadeAction"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="EditMenu"/>
   <addaction name="FormatMenu"/>
   <addaction name="windowMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="pintAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="separator"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="colorAction"/>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underLineAction"/>
   <addaction name="separator"/>
   <addaction name="leftAlignAction"/>
   <addaction name="rightAlignAction"/>
   <addaction name="centerAction"/>
   <addaction name="justifyAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建新的文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>打开一个文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>保存文档</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>文档另存</string>
   </property>
  </action>
  <action name="pintAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
   <property name="statusTip">
    <string>打印文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="pintPreviewAction">
   <property name="text">
    <string>打印预览</string>
   </property>
   <property name="statusTip">
    <string>打印预览</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出WPS</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="statusTip">
    <string>撤销这一步</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
   <property name="statusTip">
    <string>重做这一步</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪贴(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>剪贴内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="statusTip">
    <string>复制内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="statusTip">
    <string>粘贴由复制/剪贴的内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/color.ico</normaloff>:/images/color.ico</iconset>
   </property>
   <property name="text">
    <string>颜色(&amp;L)</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(&amp;B)</string>
   </property>
   <property name="statusTip">
    <string>加粗文字</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(&amp;I)</string>
   </property>
   <property name="statusTip">
    <string>倾斜文字</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="underLineAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
   </property>
   <property name="text">
    <string>下划线(&amp;U)</string>
   </property>
   <property name="statusTip">
    <string>文字下画划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="leftAlignAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐(&amp;L)</string>
   </property>
   <property name="statusTip">
    <string>段落左对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="rightAlignAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐(&amp;R)</string>
   </property>
   <property name="statusTip">
    <string>段落右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/center.png</normaloff>:/images/center.png</iconset>
   </property>
   <property name="text">
    <string>居中对齐(&amp;C)</string>
   </property>
   <property name="statusTip">
    <string>段落居中对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="justifyAction">
   <property name="icon">
    <iconset resource="QtMyWps.qrc">
     <normaloff>:/images/justify.png</normaloff>:/images/justify.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐(&amp;J)</string>
   </property>
   <property name="statusTip">
    <string>段落两端对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="closeAtion">
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有(&amp;A)</string>
   </property>
  </action>
  <action name="titleAction">
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个窗口(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>上一个窗口(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="13"/>
 <resources>
  <include location="QtMyWps.qrc"/>
 </resources>
 <connections/>
</ui>
